### PR TITLE
remove archived notebooks from the published book

### PR DESCRIPTION
**Pull request by ocefpaf**
@MathewBiddle I moved all the stale notebooks to "archived" so they don't get published anymore. Most of them are not worth revisiting but those that are, we still have the file in the repo as a starting point.

### CODE REVIEW

**Comment by MathewBiddle** [comment]
ah. Changed the extension, nice. When merged, we should check that the chapter table of contents reflect the change. For example, https://ioos.github.io/ioos_code_lab/content/code_gallery/data_access.html

**Comment by MathewBiddle** [comment]
it would be nice if you could view those notebooks in the web interface. But, this works too.

https://github.com/ocefpaf/ioos_code_lab/blob/archive_stale_notebooks/jupyterbook/content/code_gallery/data_access_notebooks/2016-12-19-exploring_csw.ipynb.archived

**Comment by ocefpaf** [comment]
> it would be nice if you could view those notebooks in the web interface

jupyterbook probably has something to ignore some notebooks in some list... Let me check if that exists before we merge this one so we don't need to rename them.

File: jupyterbook/_config.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ author: IOOS # The author of the book
 logo: images/IOOS_logo.png # A path to the book logo
 
 # Patterns to skip when building the book. Can be glob-style (e.g. "*skip.ipynb")
-exclude_patterns: [_build, Thumbs.db, .DS_Store, "**.ipynb_checkpoints"]
+exclude_patterns: [_build, Thumbs.db, .DS_Store, "**.ipynb_checkpoints", "**.archived.ipynb"]
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:7]: @MathewBiddle this should work. We only need to append the `.archived` and we can still view them on GH or nbviewer, but they won't get build as part of the book.

**Comment by ocefpaf** [comment]
This one should be ready to go. I'll fix the pre-commits and failing notebooks in #152 to avoid conflicts. Most of the failing notebooks are ERDDAP server failures that may require replacement.

File: jupyterbook/_config.yml (hunk 1 above)
Comment (MathewBiddle) [reply jupyterbook/_config.yml:7]: Awesome! Thanks for looking into this.

**Review by MathewBiddle — COMMENTED** [review]
Looks good

**Pull request merged**